lib/logstash/environment.rb

```diff
@@ -73,14 +73,6 @@ def ruby_bin
       ENV["USE_RUBY"] == "1" ? "ruby" : File.join("vendor", "jruby", "bin", "jruby")
     end
 
-    def jruby?
-      @jruby ||= !!(RUBY_PLATFORM == "java")
-    end
-
-    def windows?
-      ::Gem.win_platform?
-    end
-
```

Comment:
if the `jruby?` and `windows?` methods are defined in `lib/bootstrap/environment`, then they will be defined when `lib/logstash/environment` is required, if not then its a (easy to fix) bug. we should not have duplicate code between both.